cli/bpmetadata/types.go

```diff
@@ -68,11 +75,54 @@ type BlueprintDescription struct {
 	Architecture []string `json:"architecture,omitempty" yaml:"architecture,omitempty"`
 }
 
+type BlueprintDeploymentTime struct {
+	Configuration int `json:"configuration,omitempty" yaml:"configuration,omitempty"`
+	Deployment    int `json:"deployment,omitempty" yaml:"deployment,omitempty"`
+}
+
+type BlueprintCostEstimate struct {
+	Description string `json:",omitempty" yaml:",omitempty"`
+	Url         string `json:",omitempty" yaml:",omitempty"`
+}
+
+type BlueprintCloudProduct struct {
+	ProductId   string
+	PageUrl     string
+	Label       string
+	LocationKey bool
+}
+
+type BlueprintQuotaDetail struct {
+	QuotaType            string              `yaml:"quotaType"`
+	DynamicPropertyNames []string            `yaml:"dynamicPropertyNames,omitempty"`
+	GceInstance          GceInstanceResource `yaml:"gceInstance,omitempty"`
+	GceDisk              GceDiskResource     `yaml:"gceDisk,omitempty"`
+	DynamicResources     []struct {
+		GceInstance GceInstanceResource `yaml:"gceInstance,omitempty"`
+		GceDisk     GceDiskResource     `yaml:"gceDisk,omitempty"`
+	} `yaml:"dynamicResources,omitempty"`
+}
+
+type GceInstanceResource struct {
+	MachineType string `yaml:"machineType"`
+	Cpus        int    `yaml:"cpus"`
+}
+
+type GceDiskResource struct {
+	DiskType string `yaml:"diskType"`
+	SizeGb   int    `yaml:"sizeGb"`
+}
+
 type BlueprintMiscContent struct {
 	Name     string
 	Location string
 }
 
+type BlueprintArchitecture struct {
+	Diagram     string `json:"diagram,omitempty" yaml:"diagram,omitempty"`
```

Comment:
Is this the link? If so could this be DiagramURL?

Reply:
Changed